pkg/cookies/cookies_test.go

```diff
@@ -76,4 +79,74 @@ var _ = Describe("Cookie Tests", func() {
 			}),
 		)
 	})
+
+	Context("MakeCookieFromOptions", func() {
+		type MakeCookieFromOptionsTableInput struct {
```

Comment:
Nit, doesn't need to be public
```suggestion
		type makeCookieFromOptionsTableInput struct {
```